Formula/pokerstove.rb

```diff
@@ -0,0 +1,38 @@
+class Pokerstove < Formula
+  desc "Poker evaluation and enumeration software"
+  homepage "https://github.com/andrewprock/pokerstove"
+  url "https://github.com/andrewprock/pokerstove/archive/v1.0.tar.gz"
+  sha256 "68503e7fc5a5b2bac451c0591309eacecba738d787874d5421c81f59fde2bc74"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+
+  def install
+    mkdir "build" do
+      system "cmake", "-DCMAKE_BUILD_TYPE=Release", ".."
+      system "make"
+      bin.install "bin/ps-eval"
```

Comment:
Please use `make install` or whatever the upstream installation process is.

Reply:
The upstream didn't provide installation script at all.
```bash 
git clone https://github.com/andrewprock/pokerstove.git
mkdir pokerstove/build
cd pokerstove/build
cmake -DCMAKE_BUILD_TYPE=Release ..
make && make test
```
From [its github](https://github.com/andrewprock/pokerstove#linux).

Any suggests?

Reply:
Could we possibly do `bin.install Dir["bin/*"]` to install all files under `bin`?
Otherwise we might miss some binaries in the future, if some are added.

Reply:
Filing an issue upstream to ask them to consider a `make install` target would be welcome. This makes installation easier. For example, the github page says there is a library, but you're not installing it. Is there a reason why?